charts/auth-layer-server/README.md

```diff
@@ -0,0 +1,245 @@
+With the following configurations:
+- Realm preconfigured with the HTG (htg-auth-layer) client
+- Groups and Roles preconfigured for the HTG client
+- Custom claim mappers for the HTG client
+- Token expiration time set to 10 hours (plenty of time for a one day work session)
+- Disabled refresh tokens
+- Other custom configurations
+    - User registration is disabled
+    - Forgot password is enabled
+    - Email as username is enabled
+    - Login with email is enabled
+
+**For more information about KeyCloak Configuration, please visit the [KeyCloak Documentation](https://www.keycloak.org/docs/latest/server_admin/index.html)**
+
+## Prerequisites
+- Minikube or a Kubernetes cluster [(more here)](https://minikube.sigs.k8s.io/docs/start/)
+- Helm 3 [(install instructions here)](https://helm.sh/docs/intro/install/)
+- kubectl [(install instructions here)](https://kubernetes.io/docs/tasks/tools/)
+
+## Installation
+
+To install the Authentication Layer Server, run the following commands:
+
+- Install the chart, using the custom values file provided in the `values.yaml` file
+```bash
+helm install htg-auth-server .
+```
+
+### Helm Chart Overrides
+
+#### Admin Password
+
+Define a custom password, use `auth.adminPassword` like this: 
+```bash
+helm install htg-auth-server . --set auth.adminPassword=yourpassword
+```
+
+#### Client Secret
+
+Define a custom client secret, use `auth.clientSecret` like this: 
+```bash
+helm install htg-auth-server . --set auth.clientSecret=yourclientsecret
+```
+
+if setting the client secret from an umbrella chart and need to re-use the same client secret for other charts is recommended to use the `global.auth.clientSecret` property.
+```yaml
+global:
+    auth:
+        clientSecret: yourclientsecret
+```
+
+#### Use External DB (PostgreSQL)
+By default as part of the helm chart, it will deploy a PostgreSQL database. If you want to use an external database, you can use the following configuration:
+
+```yaml
+keycloak:
+    postgresql:
+        enabled: false
+    externalDatabase:
+        host: myexternalhost    
+        user: myuser
+        password: mypassword
+        database: mydatabase
+        port: 5432
+```
+
+
+*After installation please give some time (a few minutes) for the KeyCloak server to start and be ready to use.*
+
+*For a whole list of configurable parameters see the oficial documentation of the [KeyCloak Helm Chart from Bitnami](https://artifacthub.io/packages/helm/bitnami/keycloak/12.1.3)*
+
+### Configuration
+
+#### Port Forwarding
+You will need to port-forward the KeyCloak service to access the admin console. Run the following command to port-forward the service:
+```bash
+kubectl port-forward service/htg-auth-server-keycloak 8080:80
+```
+#### Login to the Admin Console
+Access console on the following URK: `http://localhost:8080/admin/master/console/#/HederaTheGraph`
```

Comment:
nit: this might actually be better as a link. When following the readme I ended up just copying and pasting this URL.

Reply:
✅  done